Rakefile

```diff
@@ -11,6 +11,12 @@ task :formulae do
   sh "brew", "ruby", "script/generate.rb"
 end
 
+desc "Dump cask data"
+task :cask do
```

Comment:
`task build: :dump` below needs to be updated to `task build: [:dump, :cask]`. `:dump` should arguably be renamed to `:formulae_and_analytics` if it's no longer dumping everything.

Reply:
Updated.